R/common.R

```diff
@@ -18,6 +18,12 @@ fromJSON <- function(x) jsonlite::fromJSON(x, TRUE, FALSE, FALSE)
 toJSON   <- function(x) jsonlite::toJSON(x, auto_unbox = TRUE, digits = NA, null="null")
 
 loadJaspResults <- function(name) {
+  if (base::exists(".requestStateFileNameNative")) {
+	location              <- .fromRCPP(".requestStateFileNameNative")
+	oldwd                 <- getwd()
+	setwd(location$root)
+	on.exit(setwd(oldwd))
```

Comment:
```suggestion
  if (exists(".requestStateFileNameNative")) {
    location              <- .fromRCPP(".requestStateFileNameNative")
    oldwd                 <- getwd()
    setwd(location$root)
    on.exit(setwd(oldwd))
```
fixed formatting, we prefer 2 spaces for indentation in R.